common/static/karma_common.conf.js

```diff
@@ -69,6 +70,6 @@ var options = {
     ]
 };
 
-module.exports = function (config) {
+module.exports = function(config) {
```

Comment:
We're reversing this rule in the ESLint config, this will have to go back the other way at some point 😅